integration/src/test/resources/core-namespace-routers-flow.xml

```diff
@@ -5,13 +5,31 @@
        http://www.mulesoft.org/schema/mule/core http://www.mulesoft.org/schema/mule/core/current/mule.xsd">
 
     <flow name="IdempotentSecureHashReceiverRouter">
-        <idempotent-secure-hash-message-validator messageDigestAlgorithm="SHA-128"/>
+        <idempotent-message-validator idExpression="%dw 2.0
+            output text/plain
+            import dw::Crypto
+            ---
+            Crypto::hashWith(payload,'SHA-256')"/>
```

Comment:
Actually, SHA-128 doesn't even exist. The test was just checking that the string was properly saved.